src/Components/AddTask/AddTask.tsx

```diff
@@ -155,22 +170,14 @@ const AddTask: React.FC = () => {
           </div>
           <div className="row">
             <div className="col-md-6">
-              <select
+              <Select
                 {...register("employeeId", { required: true, valueAsNumber: true })}
```

Comment:
You need to refer back to the documentation and know how to bind react-hook-form with the Select component, because it is not an HTML element.
I tried adding a user, but it seems that react-hook-form doesn't know anything about its value so it raises an error.

Other option is to get the value directly from the state and spread it in the post request after the data:
`{ ...data, employeeId: selectedUser?.value }`

but you'll need to handle the error manually.

![image](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/31bea9b2-2e54-4f88-bd06-a1d0bd3cf822)